### PR TITLE
Add basic tests for `bot.cogs.information`.

### PR DESCRIPTION
**Pull request by jchristgit**
Doesn't test `user_info` yet.

### CODE REVIEW

File: tests/cogs/test_information.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    expected = Embed(
+        title="Role information",
+        colour=Colour.blurple(),
+        description=f"`{ctx.guild.roles[0].id}` - {ctx.guild.roles[0].mention}\n"
+    )
+    expected.set_footer(text="Total roles: 1")
+
+    assert asyncio.run(coroutine) is None  # no rval
+    ctx.send.assert_called_once()
+    (_, kwargs) = ctx.send.call_args
```
Comment (scragly) [inline tests/cogs/test_information.py]:
```suggestion
    _, kwargs = ctx.send.call_args
```

File: tests/cogs/test_information.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        member('offline'), member('offline'), member('offline')
+    ]
+    ctx.guild.member_count = 1_234
+    ctx.guild.icon_url = 'a-lemon.png'
+
+    coroutine = cog.server_info.invoke(ctx)
+    assert asyncio.run(coroutine) is None  # no rval
+
+    time_since_patch.assert_called_once_with(ctx.guild.created_at, precision='days')
+    (_, kwargs) = ctx.send.call_args
```
Comment (scragly) [inline tests/cogs/test_information.py]:
```suggestion
    _, kwargs = ctx.send.call_args
```

File: tests/cogs/test_information.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    everyone_role = MagicMock()
+    everyone_role.name = '@everyone'  # should be excluded in the output
+    ctx.author.roles.append(everyone_role)
+    ctx.guild.roles = ctx.author.roles
+
+    cog.roles_info.can_run = AsyncMock()
+    cog.roles_info.can_run.return_value = True
+
+    coroutine = cog.roles_info.callback(cog, ctx)
+    expected = Embed(
```
Comment (MarkKoz) [inline tests/cogs/test_information.py]: Creating an embed object seems unnecessary if you're going to compare each field individually anyway.

**Comment by jchristgit** [comment]
yes, user_info will be a separate PR to keep these in small bits

**Pull request merged**